eng/Build.props

```diff
@@ -6,6 +6,8 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <ProjectToBuild Include="$(RepoRoot)src\Analyzers\Analyzers.sln" BuildInParallel="false" />
```

Comment:
I'm curious why build in parallel is disabled?

Reply:
I should add a comment here because it's incredibly important that this sets `BuildInParallel="false"`. Without that, it might not be built at the same time as `Razor.sln` or `Razor.Slim.slnf` which will result in build failures because `Razor.Diagnostics.Analyzers.dll` is not yet present.